src/RazorSdk/Tasks/StaticWebAssets/ComputeReferenceStaticWebAssetItems.cs

```diff
@@ -26,6 +26,8 @@ public class ComputeReferenceStaticWebAssetItems : Task
         [Required]
         public string Source { get; set; }
 
+        public bool UpdateSourceType { get; set; } = true;
```

Comment:
We now use the same logic for determining the assets to pack that we use when determining the assets that a referencing project can consume. In other words, the logic for determining the assets via ProjectReference and PackageReference is now the same.